workflow/labels.md

```diff
@@ -43,7 +44,9 @@ Each review label has 3 steps:
 | ------- | ------------------------------------------------------------------------------------------------------------------------------ |
 | Blocked | Alerts managers that unblocking it requires their help. Any blocked issue should have a comment explaining what help you need. |
 | Doing   | Actually working on the issue                                                                                                  |
-| Triaged | A team member has reviewed this issue, agreed that it is valid and assigned a priority                                         |
+| Triaged | A team member has reviewed this issue, agreed that it is valid and assigned a priority                           | Invalid | This isn't a bug. |
+| Won't fix | Even if this is a bug, we will not be fixing for a variet or reasons, like it will be replace, it isn't a priority, etc... |
```

Comment:
- It’s a constant temptation to classify issues in a way that doesn’t actually improve anyone’s decision-making. I consider it a mistake that I specified “Invalid” and “Won’t Fix” in 2005 for Launchpad’s bug tracker: total time spent by people wondering/arguing/upset about issues’ correct state far outweighed any benefit from distinguishing them. (Of course people can argue about open vs. closed too, but fewer arguable things is better.) The issue trackers in Google Code Project Hosting, then GitHub, were better by having a single closed state.
- If this was retained, `variet or` > `variety of`, `replace` > `replaced`.

Reply:
I certainly agree that people seem to like classify and categorise things which can easily end up causing a lot of busy work, and a slow, subtle drain on people's time.

I certainly agree that the distinction between "invalid" and "won't fix" doesn't seem particularly useful to anyone, and I'd happy drop one or the other. The valuable information here is that "this bug was closed without being implemented", and a single label that communicates that in an inoffensive wording as possible sounds perfectly sufficient to me (for what it's worth, I find both "invalid" and "won't fix" to be somewhat offensive, and would mind if we could find a softer word for the label instead).

Reply:
Continuing the Launchpad comparison, in 2006 when I’d realised the error of my ways, I suggested [combining “Invalid” and “Won’t Fix” into “Declined”](https://dev.launchpad.net/IssueTracker#Status) , meaning “Not appropriate or relevant for this project”.

Reply:
I like 'Declined' I will move to that and remove the others